package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@wonderlandengine/8thwall-tracking",
+  "version": "1.0.0",
+  "description": "Wonderland 8thwall implementation",
+  "main": "src/index.js",
+  "type": "module",
+  "module": "src/index.ts",
+  "scripts": {
+    "build": "tsc"
+  },
+  "keywords": [
+    "wonderland-engine"
+  ],
+  "devDependencies": {
+    "@types/qrcode-svg": "^1.1.1",
+    "typescript": "^4.9.5"
+  },
+  "dependencies": {
+    "@wonderlandengine/api": "^0.9.8",
+    "@wonderlandengine/components": "^0.9.2",
```

Comment:
I don't think this is needed? If you are using components in examples, please put a package.json there.